configs/config.msi_ms7d25_ddr4

```diff
@@ -25,6 +25,9 @@ CONFIG_PCIEXP_SUPPORT_RESIZABLE_BARS=y
 CONFIG_PCIEXP_LANE_ERR_STAT_CLEAR=y
 CONFIG_DRIVERS_EFI_VARIABLE_STORE=y
 CONFIG_DRIVERS_EFI_FW_INFO=y
+CONFIG_DRIVERS_EFI_MAIN_FW_GUID="0f3e8ba6-19d3-461b-bbbc-8b5689fe8097"
```

Comment:
Should we consider adding a check to CI that tests whether all `MAIN_FW_GUID`s are unique?

Reply:
And I guess `MAIN_FW_VERSION` could also be checked against `LOCALVERSION`.